reference/resources/gift_card_service_definition.yaml

```
@@ -0,0 +1,71 @@
+---
+title: Gift Card Service Definition
+type: object
+description: |-
+  An available gift card service that can be configured.
+x-tags:
+  - Discovery
+
+properties:
+  id:
+    type: string
+    description: The ID of the gift card service definition.
+    example: qwikcilver-gift-card
+
+  type:
+    type: string
+    description: "`gift-card-service-definition`."
+    default: gift-card-service-definition
+    example: gift-card-service-definition
```

I see this is sometimes defined as an enum to force its value:
```suggestion
  type:
    type: string
    description: "`gift-card-service-definition`."
    example: gift-card-service-definition
    enum:
      - gift-card-service-definition
```

---

Correct, will do